packages/salesforcedx-vscode-core/src/commands/functions/forceFunctionStart.ts

```diff
@@ -121,6 +121,19 @@ export class ForceFunctionStartExecutor extends SfdxCommandletExecutor<string> {
     }).execute(cancellationToken);
     const executionName = execution.command.toString();
 
+    OrgAuthInfo.getDefaultUsernameOrAlias(false)
+      .then(defaultUsernameorAlias => {
+        if (!defaultUsernameorAlias) {
+          const message = nls.localize('force_function_start_no_org_auth');
+          channelService.appendLine(message);
+          channelService.showChannelOutput();
+          notificationService.showInformationMessage(message);
```

Comment:
Do we want actionable buttons on the notification message to execute suggested commands?

Reply:
We could but I'm not aware of similar action buttons elsewhere (for example in other CLI commands, or in local dev)

@smaddox-sf @jag-sfdc We could add buttons like below:
- "Continue without Org" does nothing
- "Set a Default Org" shows the org picker - which corresponds to the first suggestion.

Note that the informational message only appears for a few seconds and disappear, unless it's displayed as a modal prompt. Thoughts?

<img width="460" alt="Screen Shot 2020-09-21 at 3 43 14 PM" src="https://user-images.githubusercontent.com/679275/93831734-6022b480-fc28-11ea-8642-abca3af6861d.png">

Reply:
For uniform experience and also to keep it simple for beta, I think we should skip this.

Reply:
Agreed.  I thought we were just going to output a message in the 'output' panel telling them no default org set.  (just as an FYI, no prompt)

Reply:
@smaddox-sf - tell me what you decide in case I don't need to edit this message.